src/EditorFeatures/CSharpTest/EditAndContinue/RudeEditStatementTests.cs

```diff
@@ -4416,7 +4416,7 @@ void F()
         }
 
         // Add corresponding test to VB
-        [WpfFact(Skip = "TODO")]
+        [Fact(Skip = "TODO")]
```

Comment:
Could you please change the skip string to `"https://github.com/dotnet/roslyn/pull/18940"`